### ISSUE TITLE
BUG - Kotlin boolean variable

### ISSUE THREAD

**Issue by recoverrelax** [evernote/android-state#18]
i have this fields in a presenter:

```
@State var firsTime: Boolean = false
@State var isFlavorGo: Boolean = false
```

And the annotation processor gives me this error:

> error: Field must be either non-private or provide a getter and setter method
> e:     private boolean isFlavorGo;
> e:                     ^

**Comment by recoverrelax** [comment]
- found put the problem. If a variable is declared like this (in kotlin)
`@State var isFlavorGo: Boolean = false`
the generated setter is like this: 
`setFlavorGo()` and not 'setIsFlavorGo'. Than must mess with the annotation processor.

Anyway, it is a bug. Fixed it is simple, change the name `isFlavorGo` to `flavorGo`

**Comment by recoverrelax** [comment]
was checking the change log, apparently the bug was already fixed. I'm using compile 

> 'com.evernote:android-state:1.0.5' and the bug still there (Kotlin)

**Comment by vRallev** [comment]
No, I missed that case. Thanks for taking a longer at it.

**Comment by vRallev** [comment]
It'll be fixed in the next version. I'd appreciate if you give the snapshot build a try
```groovy
repositories {
    jcenter()

    maven {
        url 'https://oss.sonatype.org/content/repositories/snapshots/'
    }
}

dependencies {
    compile 'com.evernote:android-state:1.1.0-SNAPSHOT'
    kapt 'com.evernote:android-state-processor:1.1.0-SNAPSHOT'
}
```

**Comment by recoverrelax** [comment]
yes. it fixes the issue with the 1.1.0-SNAPSHOT.
Thanks for the fast feedbacl
